blitz_api/views.py

```diff
@@ -603,22 +548,8 @@ class AcademicFieldViewSet(viewsets.ModelViewSet):
     permission_classes = (permissions.IsAdminOrReadOnly,)
     ordering = ('name',)
 
-    @action(detail=False, permission_classes=[IsAdminUser])
-    def export(self, request):
-        # Use custom paginator (by page, min/max 1000 objects/page)
-        self.pagination_class = ExportPagination
-        # Order queryset by ascending id, thus by descending age too
-        queryset = self.get_queryset().order_by('pk')
-        # Paginate queryset using custom paginator
-        page = self.paginate_queryset(queryset)
-        # Build dataset using paginated queryset
-        dataset = AcademicFieldResource().export(page)
-        # Build response object
-        response = self.get_paginated_response(dataset.xls)
-        # Add filename to response
-        response['Content-Disposition'] = ''.join([
-            'attachment; filename="AcademicField-',
-            LOCAL_TIMEZONE.localize(datetime.now()).strftime("%Y%m%d-%H%M%S"),
-            '".xls'
-        ])
-        return response
+
+class ExportMediaViewSet(viewsets.ModelViewSet):
```

Comment:
Maybe a permission to limit access to admin only

Reply:
Done!
```suggestion
class ExportMediaViewSet(viewsets.ModelViewSet):
```
And I also added new factories